lib/smart_answer/calculators/benefit_cap_calculator_configuration.rb

```diff
@@ -46,12 +46,14 @@ def region(postcode)
     end
 
     def london?(postcode)
-      area(postcode).any? { |result| result[:type] == 'EUR' && result[:name] == "London" }
+      area(postcode).any? do |result|
```

Comment:
Please, update this commit with the OpenStruct direct access once we are using the OpenStruct response object from the GdsAdapters.